cli/src/pcluster/templates/cdk_builder_utils.py

```diff
@@ -51,26 +49,18 @@ def get_block_device_mappings(node_config: Union[HeadNode, BaseQueue], os: str):
             ec2.CfnLaunchTemplate.BlockDeviceMappingProperty(device_name=device_name, virtual_name=virtual_name)
         )
 
-    # Root volume
-    if isinstance(node_config, BaseQueue):  # Queue
-        if (
-            node_config.compute_settings
-            and node_config.compute_settings.local_storage
-            and node_config.compute_settings.local_storage.root_volume
-        ):
-            root_volume = copy.deepcopy(node_config.compute_settings.local_storage.root_volume)
-        else:
-            root_volume = Ebs()
-    elif isinstance(node_config, HeadNode):  # HeadNode
-        if node_config.local_storage and node_config.local_storage.root_volume:
-            root_volume = copy.deepcopy(node_config.local_storage.root_volume)
-        else:
-            root_volume = Ebs()
+    root_volume = local_storage.root_volume or Ebs()
```

Comment:
Should we keep the deep_copy ?

Reply:
Double checked, I don't think it is necessary